src/pages/CreateNewCourseForm.tsx

```diff
@@ -0,0 +1,126 @@
+import React from "react";
+import Box from "@mui/material/Box";
+import { BoxTitle } from "../design/typography";
+import Navbar from "../components/Navbar/Navbar";
+import { Grid, IconButton, Paper } from "@mui/material";
+import Button from '@mui/material/Button';
+import ArrowBackIosIcon from '@mui/icons-material/ArrowBackIos';
+import { useNavigate } from "react-router-dom";
+import Layout from "../components/shared/Layout";
+
+
+/*type NewCourseForm = {
+    content: number;
+    teaching:number;
+    workload: number;
+    text: string;
+  };*/
```

Comment:
do we want to keep this?

Reply:
yes, for the form, which is not yet finished and I wanted to push, so that we do not have the errors anymore :)